projects/core/utils/format/number-to-string-without-exp.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Convert number to string with replacing exponent part on decimals
+ *
+ * @param value the number
+ * @return string representation of a number
+ */
+export function numberToStringWithoutExp(value: number): string {
+    const valueAsString = value.toString();
+    const [numberPart, expPart] = valueAsString.split('e-');
+
+    let valueWithoutExp = valueAsString;
+
+    if (expPart) {
+        const [, fractionalPart] = numberPart.split('.');
+        const decimalDigits = Number(expPart) + (fractionalPart?.length ?? 0);
```

Comment:
Let's change `??` to `||`. It makes no difference here but `||` is transpiled into simpler code.

Reply:
done